src/main/java/uk/gov/hmcts/befta/dse/ccd/TestDataLoaderToDefinitionStore.java

```diff
@@ -113,23 +113,44 @@ protected void addCcdRole(CcdRoleConfig roleConfig) {
 
     protected List<String> getAllDefinitionFilesToLoad() {
         try {
+            boolean convertJsonFilesToExcel = false;
+            Set<String> definitionJsonResourcesToTransform = new HashSet<>();
             List<String> definitionFileResources = new ArrayList<String>();
             ClassPath cp = ClassPath.from(Thread.currentThread().getContextClassLoader());
             for (ClassPath.ResourceInfo info : cp.getResources()) {
                 if (info.getResourceName().startsWith(definitionsPath)
                         && info.getResourceName().toLowerCase().endsWith(".xlsx")
                         && !info.getResourceName().startsWith("~$")) {
                     definitionFileResources.add(info.getResourceName());
+                } else if (info.getResourceName().startsWith(definitionsPath)
+                        && info.getResourceName().toLowerCase().endsWith(".json")){
+                    convertJsonFilesToExcel = true;
+                    File jsonFile = BeftaUtils.createJsonDefinitionFileFromClasspath(info.getResourceName());
+                    String jsonDefinitionParentFolder = jsonFile.getParentFile().getParentFile().getPath();
+                    definitionJsonResourcesToTransform.add(jsonDefinitionParentFolder);
                 }
             }
+
+            if (convertJsonFilesToExcel) {
+                definitionFileResources.addAll(
+                        definitionJsonResourcesToTransform.stream()
+                        .map(folderPath -> new JsonTransformer(folderPath,"build/tmp").transformToExcel())
+                        .collect(Collectors.toList()));
+            }
```

Comment:
Hmm am not too sure this would work as line 130 is in a loop of class path resources so this would be trying to create a definition file each time it finds a single json file